tests/sfp_test.py

```diff
@@ -127,3 +275,83 @@ def teardown_class(cls):
         print("TEARDOWN")
         os.environ["PATH"] = os.pathsep.join(os.environ["PATH"].split(os.pathsep)[:-1])
         os.environ["UTILITIES_UNIT_TESTING"] = "0"
+        os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = ""
+
+class Test_multiAsic_SFP(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        os.environ["PATH"] += os.pathsep + scripts_path
+        os.environ["UTILITIES_UNIT_TESTING"] = "2"
+        os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = "multi_asic"
+
+    def test_sfp_presence_with_ns(self):
+        runner = CliRunner()
+        result = runner.invoke(show.cli.commands["interfaces"].commands["transceiver"].commands["presence"], ["Ethernet0 -n asic0"])
```

Comment:
>["Ethernet0 -n asic0"] [](http://example.com/codeflow?start=109&length=22)

I think this is incorrect way to parse arguments. Ref: [https://click.palletsprojects.com/en/8.1.x/testing/#id4](https://click.palletsprojects.com/en/8.1.x/testing/#id4)